src/OrchardCore/OrchardCore.Users.Core/OrchardCore.Users.Core.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\OrchardCore.DisplayManagement.Abstractions\OrchardCore.DisplayManagement.Abstractions.csproj" />
```

Comment:
Need this reference to have `IUpdateModel` 

Need `IUsersAdminListFilter` in the Core project because `User` is in the core project.

Wonder about moving `User` to the abstractions project, to avoid this `DisplayManagement.Abstractions` reference.